README.md

```diff
@@ -3,13 +3,13 @@
 [![License](https://poser.pugx.org/scienta/doctrine-json-functions/license)](https://packagist.org/packages/scienta/doctrine-json-functions)
 
 # DoctrineJsonFunctions
-A set of extensions to Doctrine 2 that add support for json query functions.
+A set of extensions to Doctrine 2+ that add support for json functions.
 +Functions are available for MySQL, MariaDb and PostgreSQL.
 
 | DB | Functions |
 |:--:|:---------:|
 | MySQL | `JSON_APPEND, JSON_ARRAY, JSON_ARRAYAGG, JSON_ARRAY_APPEND, JSON_ARRAY_INSERT, JSON_CONTAINS, JSON_CONTAINS_PATH, JSON_DEPTH, JSON_EXTRACT, JSON_OVERLAPS, JSON_INSERT, JSON_KEYS, JSON_LENGTH, JSON_MERGE, JSON_MERGE_PRESERVE, JSON_MERGE_PATCH, JSON_OBJECT, JSON_OBJECTAGG, JSON_PRETTY, JSON_QUOTE, JSON_REMOVE, JSON_REPLACE, JSON_SEARCH, JSON_SET, JSON_TYPE, JSON_UNQUOTE, JSON_VALID` |
-| PostgreSQL | `JSON_EXTRACT_PATH, GT, GT_GT, SHARP_GT, SHARP_GT_GT` |
+| PostgreSQL | `JSONB_CONTAINS, JSONB_EXISTS, JSONB_EXISTS_ALL, JSONB_EXISTS_ANY, JSONB_IS_CONTAINED, JSON_EXTRACT_PATH, JSON_GET, JSON_GET_PATH, JSON_GET_PATH_TEXT, JSON_GET_TEXT` |
```

Comment:
i could not find any of those `GT` functions, i assume they got renamed to GET

Reply:
These are references to the specific postgresql functions (non-DQL): https://www.postgresql.org/docs/9.5/functions-json.html

SHARP_GT_GT stands for `#>>`.
GT_GT stands for `->>` if i'm correct.

Your edits now refer to the DQL implementation, which is not what the README tries to explain here.

It could be clarified by using the notation from the docs.
Can you update your PR to reflect that?

Reply:
ah, i see. for the JSONB_ functions, the named functions actually exist in postgres as well (even though i can't find them referenced in the postgres documentation). how about i list each operator and in paranthesis what it is called in this library?

without having both, its difficult to know what to use to achieve a specific operation.

Reply:
`@> (JSONB_CONTAINS), ? (JSONB_EXISTS), ?& (JSONB_EXISTS_ALL), ?| (JSONB_EXISTS_ANY), <@ (JSONB_IS_CONTAINED), JSON_EXTRACT_PATH, -> (JSON_GET), #> (JSON_GET_PATH), #>> (JSON_GET_PATH_TEXT), ->> (JSON_GET_TEXT)`

Reply:
that would be nice!